.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: required
+
+language: ruby
+
+services:
```

Comment:
Better avoid this, since you install Docker manually in the next steps.

Reply:
Language ruby is the dafault, avoiding this caused a validation warning in  https://lint.travis-ci.org

Reply:
What about using `language: generic` ?

See:

* https://github.com/travis-ci/travis-ci/issues/4895
* https://github.com/travis-ci/travis-ci/issues/2867#issuecomment-59262795

Reply:
OK 392b263c446f22b75216c21f908c29cd4c33fc1c

Reply:
I was more referring to the Docker service, which feels unnecessary here since you install it manually.